src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/AccessToken/OidcTokenHandlerFactory.php

```diff
@@ -155,19 +128,11 @@ public function addConfiguration(NodeBuilder $node): void
                         ->isRequired()
                         ->scalarPrototype()->end()
                     ->end()
-                    ->arrayNode('algorithm')
-                        ->info('Algorithm used to sign the token.')
-                        ->setDeprecated('symfony/security-bundle', '7.1', 'The "%node%" option is deprecated and will be removed in 8.0. Use the "algorithms" option instead.')
-                    ->end()
                     ->arrayNode('algorithms')
```

Comment:
we need `fixXmlConfig('algorithm')` to convert the single to plural for the XML format for the prototyped node